circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   python:
-    version: 3.4.2
+    version: 3.5.0
   services:
     - postgresql
 dependencies:
```

Comment:
Down below is `pip-accel` which I believe has been removed from the template - I think it can just be removed